cmd/load-generator/main.go

```diff
@@ -0,0 +1,272 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/codegangsta/cli"
```

Comment:
This command will have to be rewritten without this import - master has removed it.

```
cmd/load-generator/main.go:11:2: cannot find package "github.com/codegangsta/cli" in any of:
	/go/src/github.com/letsencrypt/boulder/vendor/github.com/codegangsta/cli (vendor tree)
	/usr/local/go/src/github.com/codegangsta/cli (from $GOROOT)
	/go/src/github.com/codegangsta/cli (from $GOPATH)
```